synapse/config/server.py

```diff
@@ -490,6 +500,20 @@ def default_config(self, server_name, data_dir_path, **kwargs):
 
         # Used by phonehome stats to group together related servers.
         #server_context: context
+
+        # Using SQLite with Synapse induces a severe degradation in performances
+        # compared to using PostgreSQL. SQLite should only be used for personal
+        # testing. Production instances should use PostgreSQL instead. You can
+        # find documentation on setting up PostgreSQL with Synapse here:
+        # https://github.com/matrix-org/synapse/blob/master/docs/postgres.rst
+        #
+        # If you want to use federation with SQLite regardless, you can
+        # uncomment the line below, but be aware that it can make Synapse
```

Comment:
I think this is overdoing things a bit. We've already said it will cause a performance impact, and it shouldn't make it *malfunction*. I'd remove the part starting at `but`.